pkg/parsers/filters/parse.go

```diff
@@ -36,9 +37,10 @@ func ParseFlag(arg string, prev Args) (Args, error) {
 	return filters, nil
 }
 
+// ErrorBadFormat bad format for filter
```

Comment:
How about "ErrorBadFormat is an error returned in case of a bad format for a filter"?